### PR TITLE
polygon v2 reserve factor update april

### PR DESCRIPTION
**Pull request by dd0sxx**
ready for review. when ready we can open a pr against the bgd repo

### CODE REVIEW

File: src/20240322_AaveV2Polygon_ReserveFactorUpdates/AaveV2Polygon_ReserveFactorUpdates_20240322.sol [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV2Polygon, AaveV2PolygonAssets} from 'aave-address-book/AaveV2Polygon.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+/**
+ * @title Reserve Factor Updates (March 15, 2024)
```
Comment (efecarranza) [inline src/20240322_AaveV2Polygon_ReserveFactorUpdates/AaveV2Polygon_ReserveFactorUpdates_20240322.sol:8]: Update title

File: src/20240322_AaveV2Polygon_ReserveFactorUpdates/AaveV2Polygon_ReserveFactorUpdates_20240322.t.sol [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ */
+contract AaveV2Polygon_ReserveFactorUpdates_20240322_Test is ProtocolV2TestBase {
+  struct Changes {
+    address asset;
+    uint256 reserveFactor;
+  }
+  AaveV2Polygon_ReserveFactorUpdates_20240322 internal proposal;
+
+  function setUp() public {
+    vm.createSelectFork(vm.rpcUrl('polygon'), 54963727);
```
Comment (efecarranza) [inline src/20240322_AaveV2Polygon_ReserveFactorUpdates/AaveV2Polygon_ReserveFactorUpdates_20240322.t.sol:21]: Update block as previous RF had not executed

File: src/20240322_AaveV2Polygon_ReserveFactorUpdates/ReserveFactorUpdates.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+---
+title: "ReserveFactorUpdates"
+author: "TokenLogic"
+discussions: "https://hackmd.io/VuOFSRuzSXOb0vNxP5-Otw"
+---
+
+## Simple Summary
+
+This AIP is a continuation of proposal 41 on Governance v3 and increases the Reserve Factor (RF) for assets on Polygon v2 by 5.00%, up to a maximum of 99.99%.
```
Comment (efecarranza) [inline src/20240322_AaveV2Polygon_ReserveFactorUpdates/ReserveFactorUpdates.md:9]: Continuation of 54

File: diffs/AaveV2Polygon_ReserveFactorUpdates_20240322_before_AaveV2Polygon_ReserveFactorUpdates_20240322_after.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+## Reserve changes
+
+### Reserve altered
+
+#### USDC ([0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174](https://polygonscan.com/address/0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174))
+
+| description | value before | value after |
+| --- | --- | --- |
+| reserveFactor | 83 % | 93 % |
```
Comment (efecarranza) [inline diffs/AaveV2Polygon_ReserveFactorUpdates_20240322_before_AaveV2Polygon_ReserveFactorUpdates_20240322_after.md:9]: Rerun tests so these update

**Review by efecarranza — CHANGES_REQUESTED** [review]

**Comment by dd0sxx** [comment]
closing because this was already merged into the bgd repo

**Pull request closed without merging**